CHANGELOG

```diff
@@ -1,18 +1,29 @@
 2.9.0.dev
----------
+=========
+
+New Features
+------------
 
 * New `pytest.mark.skip` mark, which unconditional skips marked tests.
   Thanks Michael Aquilina for the complete PR.
 
-* fix issue #680: the -s and -c options should now work under xdist;
-  `Config.fromdictargs` now represents its input much more faithfully.
-  Thanks to Buck Evan for the complete PR.
+* New `-rp` and `-rP` reporting options give the summary and full output
+  of passing tests, respectively. Thanks to David Vierra for the PR.
+
+
+Changes
+-------
 
 * `pytest_enter_pdb` now optionally receives the pytest config object.
   Thanks Bruno Oliveira for the PR.
 
-* New `-rp` and `-rP` reporting options give the summary and full output
-  of passing tests, respectively. Thanks to David Vierra for the PR.
+
+Bug Fixes/Adjustments
```

Comment:
Hmm in retrospect, I think this should just be "Bug Fixes", and the #1226 mention in this section should be moved to "Changes". As it stands now, "Adjustments" and "Changes" is ambiguous (after all, it seems an "adjustment" is just a "small change", which is not good IMO), and even an internal refactoring is a change after all (as much as we would like for them to have no impact at all, it might just happen some corner case creeps up).

But I will fix this in a later commit, unless you disagree.